moler/device/textualdevice.py

```diff
@@ -35,7 +35,8 @@ class TextualDevice(object):
     not_connected = "NOT_CONNECTED"
     connection_hops = "CONNECTION_HOPS"
 
-    def __init__(self, sm_params=None, name=None, io_connection=None, io_type=None, variant=None, initial_state=None):
+    def __init__(self, sm_params=None, name=None, io_connection=None, io_type=None, variant=None,
+                 io_constructor_kwargs={}, initial_state=None):
```

Comment:
New argument as last one?

Reply:
I think is better when all io_connection parameters are together.